examples/navigation/epics/adminAccess.js

```diff
@@ -0,0 +1,15 @@
+import { push, LOCATION_CHANGE } from 'react-router-redux';
+
+function hasAccess() {
+  return false;
+}
+
+export default function adminAccess(action$) {
+  return action$.ofType(LOCATION_CHANGE)
+    .map(action => action.payload.pathname)
+    .filter(pathname => pathname === '/admin')
+    .map(hasAccess)
+    .filter(access => !access)
```

Comment:
Sorry, I should have left a comment. Those dummy lines are to imply that more meaningful code should go there to *actually* check for access – it's just for semantics. I could leave a comment to the existing code or just omit it all together.